app/views/nd_certificates/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="mt-3 card bg-off-white rounded-0 border-0 mb-4">
+  <%= link_to condo_path(@condo.id), class: "ms-3 mt-3" do %>
+    <%= image_tag "arrow-left", size: "25", id: "arrow-left" %>
+  <% end %>
+
+  <span class="fs-4 mx-auto mt-3 text-uppercase"> <%= I18n.t('units') %> </span>
+  <span class="mx-auto text-uppercase fs-7 text-muted"><%= @condo.name %></span>
+
+  <div class="d-flex justify-content-center mt-4 mb-4">
+    <% if @units.any? %>
+      <div class="list-group w-75">
+        <% @units.each do |unit| %>
+          <%= link_to condo_nd_certificate_path(@condo.id, unit.id), class: "list-group-item p-3 list-group-item-action hover-strong", data: { turbo_frame: "modal" } do %>
+            <div class="d-flex w-100 justify-content-between">
+              <span class="d-block mb-1 fs-5">
+                 <%=I18n.t('views.index.unit')%> <%= unit.number %>
+              </span>
+              <div class="d-flex flex-wrap text-end">
+                <small class="text-body-secondary w-100">
+                  ID:
+                  <%= unit.id%>
```

Comment:
Porque mostrar o ID da unidade para o usuário? Não sei se entendi... e se for mostrar o id, seria bom dar um espaço antes de fechar a tag.